Calendar/Day.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import { TouchableOpacity, Text, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    padding: 10,
+  },
+});
+
+type Props = {
+  date: Date,
+  currentMonth: number,
+  renderDate: React.Node,
+  onSelect: () => void,
+}
+
+function Day({ date, currentMonth, renderDate, onSelect }: Props) {
+  const dateObj = new Date(date);
+  return (
+    <TouchableOpacity
+      style={styles.container}
+      onPress={() => {
+        if (onSelect) {
+          onSelect(date);
+        }
+      }}
+    >
+      <Text
+        allowFontScaling={false}
+        style={{ color: currentMonth === dateObj.getMonth() ? 'black' : 'grey' }}
```

Comment:
I think we should keep the colors flexible. Get it from the props and use default values.